README.md

```diff
@@ -52,6 +53,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 ## S - U
 - [SPI Nepal](http://www.softvision.com/) | Thapathali, Kathmandu | Provider of Digital Solutions, Mobility, Analytics, Merchandise Planning, In-Store Engagements and Quality Assurance
```

Comment:
- [Satyam Multi-Media House] (https://smmh.com.np) | Dillibazar, Kathmandu | Our specialization is in software, website, and app development along with its marketing. We create innovative and cutting-edge solutions to help businesses stay ahead in the digital world.